### ISSUE TITLE
Brew formula is not updated to 0.2.0 (still 0.2.0-rc4)

### ISSUE THREAD

**Issue by dio** [tetratelabs/getenvoy#103]
Seems like the brew formula still points to [0.2.0-rc4](https://github.com/tetratelabs/homebrew-getenvoy/blob/60962580b32e616f551944dd0c1e7be9b850f282/Formula/getenvoy.rb#L9).

```
$ brew install getenvoy
Updating Homebrew...
==> Installing getenvoy from tetratelabs/getenvoy
==> Downloading https://github.com/tetratelabs/getenvoy/releases/download/0.2.0-rc4/getenvoy_0.2.0-rc4_Darwin_x86_64.tar.gz
Error: Failed to download resource "getenvoy"
```

**Comment by yskopets** [comment]
Yeah, sorry.

I released `0.2.0-rc4` after `0.2.0` and it became the latest version.

I will rollback `0.2.0-rc4` release.

**Comment by yskopets** [comment]
@dhi Please see https://github.com/tetratelabs/homebrew-getenvoy/pull/26
